frontend/public/components/monitoring/receiver-forms/routing-labels-editor.tsx

```diff
@@ -0,0 +1,162 @@
+        <div className="col-xs-5">Name</div>
+        <div className="col-xs-6">Value</div>
+      </div>
+      {isDefaultReceiver && (
+        <div className="row form-group" key="default">
+          <div className="col-xs-10">
+            <div className="row">
+              <div className="col-xs-6 pairs-list__name-field">
+                <div className="form-group">
+                  <input
+                    type="text"
+                    className="pf-c-form-control"
+                    data-test-id={`label-name-default`}
+                    value={DEFAULT_RECEIVER_LABEL}
+                    disabled
+                    required
+                  />
+                </div>
+              </div>
+              <div className="col-xs-6 pairs-list__value-field">
+                <div className="form-group">
+                  <input
+                    type="text"
+                    className="pf-c-form-control"
+                    data-test-id={`label-value-default`}
+                    value={DEFAULT_RECEIVER_LABEL}
+                    disabled
+                    required
+                  />
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      )}
+      {_.map(formValues.routeLabels, (routeLabel, i) => {
+        return (
+          <div className="row form-group" key={i}>
+            <div className="col-xs-10">
+              <div className="row">
+                <div className="col-xs-6 pairs-list__name-field">
+                  <div className="form-group">
+                    <input
+                      type="text"
+                      className="pf-c-form-control"
+                      data-test-id={`label-name-${i}`}
+                      onChange={onRoutingLabelChange(`${i}, name`)}
+                      placeholder="Name"
+                      value={routeLabel.name}
+                      required
+                    />
+                  </div>
+                </div>
+                <div className="col-xs-6 pairs-list__value-field">
+                  <div className="form-group">
+                    <input
+                      type="text"
+                      className="pf-c-form-control"
+                      data-test-id={`label-value-${i}`}
+                      onChange={onRoutingLabelChange(`${i}, value`)}
+                      placeholder="Value"
+                      value={routeLabel.value}
+                      required
+                    />
+                  </div>
+                </div>
+              </div>
+              <div className="row">
+                <div className="col-xs-12 col-sm-12">
+                  <div className="form-group">
+                    <label className="co-no-bold">
+                      <input
+                        type="checkbox"
+                        onChange={(e) => onRoutingLabelRegexChange(e, _.toNumber(i))}
```

Comment:
Isn't `i` already a number?

Reply:
yes, just had to add an initial type of `: number`.  fixed